pkg/gofr/datasource/sql/sql.go

```diff
@@ -65,10 +77,16 @@ func NewSQL(configs config.Config, logger datasource.Logger, metrics Metrics) *D
 		return database
 	}
 
+	logger.Debugf("checking database '%s' connection", database.config.Database)
+
 	database = pingToTestConnection(database)
 
+	logger.Debug("enabling retry logic incase of failures/errors")
+
 	go retryConnection(database)
 
+	logger.Debug("pushing DB metrics to opentelemetry")
+
```

Comment:
IMO, these logs are not relevant as we already have subsequent logs in respective functions.